src/main/java/io/fabric8/elasticsearch/plugin/acl/DynamicACLFilter.java

```diff
@@ -65,56 +80,78 @@
  */
 public class DynamicACLFilter 
 	extends RestFilter 
-	implements ConfigurationSettings, SearchGuardACLActionRequestListener {
+	implements ConfigurationSettings {
 
 	private static final String AUTHORIZATION_HEADER = "Authorization";
-	private static final String SEARCHGUARD_TYPE = "ac";
-	private static final String SEARCHGUARD_ID = "ac";
-	private static final String[] BLACKLIST_PROJECTS = {"default", "openshift", "openshift-infra" };
+	private static final String[] BLACKLIST_PROJECTS = {"default", "openshift", "openshift-infra", "kube-system" };
```

Comment:
it should be.. i mistakenly thought we had that in master; i'll update to add that in i believe i have it stashed in another branch